### ISSUE TITLE
Metabolic Alkalosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5872]
**HPO term**

https://hpo.jax.org/app/browse/term/HP:0200114
**Suggested revision of textual definition**

Proposed definition: Alkalosis due to loss of acids or accumlation of base

**Comment by pnrobinson** [comment]
adding definition from https://pubmed.ncbi.nlm.nih.gov/29493916/
